website/client/app.vue

```diff
@@ -104,9 +104,8 @@ div
 <style lang='scss'>
   @import '~client/assets/scss/colors.scss';
 
-  /* @TODO: The modal-open class is not being removed. Let's try this for now */
   .modal {
-    overflow-y: scroll !important;
+    overflow-y: auto !important;
```

Comment:
Can you revert this. We still need the overflow hack

Reply:
oh, it just looked ugly on windows :D and I thought if I change it to auto then it will be visible if needed

Reply:
Yea, it's a hack :P Can you add the note back too? You have PR for fixing modals, so after that is merged in we can test removing it.